src/steps/install-dependencies.test.ts

```diff
@@ -13,6 +13,7 @@ let execa: jest.Mock;
 jest.mock('../utils/print-header');
 jest.mock('../utils/get-context');
 jest.mock('../utils/package-manager');
+jest.mock('../utils/lockfile-manager.ts');
```

Comment:
Let’s make this the same as the other lines:

```suggestion
jest.mock('../utils/lockfile-manager');
```